locopy/snowflake.py

```diff
@@ -468,7 +468,7 @@ def to_dataframe(self, size=None):
             Dataframe with lowercase column names.  Returns None if no fetched
             result.
         """
-        if size is None:
+        if size is None and self.cursor._query_result_format == 'arrow':
```

Comment:
So this now excludes all cases where `size` was none that were not in `arrow`-format. Those were the ones that are the result of an operational statement?

Reply:
Yes, if you look at the page I shared from snowflake connector, and I also tested with several queries. Operational statements would have a Json-format.

Reply:
Ah, makes sense